### DISCUSSION TITLE
Setup for Adonis and Quasar

### DISCUSSION THREAD

**Discussion by rbartholomay** [adonisjs/core#4043] [Help]
Did someone use the combination of Adonisjs with Quasar?   I need some hints how to setup this correctly. (Directoy-Structure etc.)

Thanks a lot...

**Comment by Melchyore** [comment]
@rbartholomay Hey!

I plan to use this combination for the new website lf my company. The best advice I can give you is to use 2 separated apps: one for the frontend (Quasar framework) and one for the backend (AdonisJS) and Nginx as a reverse-proxy. You should use SSR on your Quasar app to have a better SEO. Using 2 apps will also allow you to create a mobile app (if planned for the future) as you can do it with Quasar. So, use 2 folders or repos, it's up to you, but don't merge them.

**Comment by rbartholomay** [comment]
Hi!

i use the combination (adonis, Quasar) in the past to have seperate apps decoupled from each other. This runs fine, also in development. Open two terminals, one for adonis development and another for the Quasar-Demove via vite.

But i need hint how to setup the combination if i need to use "inside" adonisjs. I now that it mus running via encore, but what is the right way?
